x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/support/ApiKeyFieldNameTranslators.java

```diff
@@ -302,6 +302,157 @@ private static Set<String> translatePattern(String fieldNameOrPattern) {
         return indexFieldNames;
     }
 
+    public static QueryBuilder translateQueryFields(QueryBuilder qb, @Nullable Consumer<String> visitor) {
```

Comment:
This is just a merge issue right? We should be able to use `translateQueryBuilderFields` instead -- the two methods are identical apart from a few missing comments/var names.

Reply:
Yes!!!
Good catch!

Reply:
Pushed [b7bd8fc](https://github.com/elastic/elasticsearch/pull/104895/commits/b7bd8fc2fa3befaa44e89977b4f6f27f8abf6fd1)